spec/public_organization_exporter_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+require 'public_organization_exporter'
+
+describe PublicOrganizationExporter do
+  let(:exporter) { PublicOrganizationExporter.new }
+
+  describe '#headers' do
+    it "generates localized headers" do
+      expect(exporter.headers.first).to eq('referencia')
+      expect(exporter.headers.last).to eq('aproximación')
+    end
+  end
+
+  describe '#public_organization_to_row' do
+    it "generates a row of info based on a public organization" do
+      o = create(:organization)
+      row = exporter.organization_to_row(o)
+
+      expect(row).to include(o.reference)
+      expect(row).to include(o.identifier)
+      expect(row).to include(o.address_type)
+      expect(row).to include(o.address)
+      expect(row).to include(o.number)
+      expect(row).to include(o.stairs)
+      expect(row).to include(o.floor)
+      expect(row).to include(o.door)
+      expect(row).to include(o.postal_code)
+      expect(row).to include(o.town)
+      expect(row).to include(o.province)
+      expect(row).to include(o.phones)
+      expect(row).to include(o.email)
+      # expect(row).to include(o.association)
```

Comment:
We can't export this right now, but we'll be able soon? I mean, why is this line commented out? If its a task we can't do right now, I would better create an issue in this repo than leave a comment on the code

Reply:
it's an old code that I forgot to erase